developer/contributing/pull-requests.rst

```diff
@@ -23,21 +29,22 @@ When creating a pull request:
  * **review**: when it is reviewable
  * **feedback**: if you have reviewed someone else's pr
 
-We recommend that you can also use the `Waffle board`_ to manage the state of your
-PRs and issues.
+We recommend that you can also use the `Waffle board`_ to manage the state of
+your PRs and issues.
```

Comment:
The same is valid for assigning labels... I guess we should extract that into an own section for organization members.